Gemfile

```diff
@@ -15,12 +15,10 @@ gem 'traject_alephsequential_reader'
 gem 'traject_umich_format'
 gem 'date_named_file'
 
-if defined? JRUBY_VERSION
-  gem 'jdbc-mysql'
-  gem 'traject-marc4j_reader'
-#  gem 'jruby-launcher', '~>1.1'
-#  gem 'naconormalizer', path: '../naconormalizer'
-end
+gem 'jdbc-mysql', platforms: :jruby
+gem 'traject-marc4j_reader', platforms: :jruby
```

Comment:
I got some strange errors from Bundler with the previous version about `traject-marc4j_reader` -- in particular issues about there being no eligible candidates to install.